clang/lib/CodeGen/CGCall.cpp

```diff
@@ -1861,6 +1861,8 @@ void CodeGenModule::getDefaultFunctionFPAccuracyAttributes(
         ID, FuncType, convertFPAccuracy(M.second));
     if (!FPAccuracyVal.empty())
       FuncAttrs.addAttribute("fpbuiltin-max-error=", FPAccuracyVal);
+    else
```

Comment:
I don't like the "else assert" here. I'm not sure it will even compile correctly in non-assert builds.
```suggestion
   assert(!FPAccuracyVal.empty() && "A valid accuracy value is expected");
   FuncAttrs.addAttribute("fpbuiltin-max-error", FPAccuracyVal);
```